src/SFA.DAS.AssessorService.Web.UnitTests/CertificateTests/When_submitting_passed_certificate_with_no_option_selected.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FizzWare.NBuilder;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using SFA.DAS.AssessorService.Domain.Consts;
+using SFA.DAS.AssessorService.Domain.Entities;
+using SFA.DAS.AssessorService.Domain.JsonData;
+using SFA.DAS.AssessorService.Web.Controllers;
+using System;
+using Organisation = SFA.DAS.AssessorService.Domain.Entities.Organisation;
+
+namespace SFA.DAS.AssessorService.Web.UnitTests.CertificateTests
+{
+    public class When_submitting_passed_certificate_with_no_option_selected : CertificateCheckControllerTestBase
+    {
+        private ViewResult _result;
+
+        [SetUp]
+        public void Arrange()
```

Comment:
I really don't like all the setup and config in the existing unit test arrangements. When I've been making changes, i've been standardising the unit tests, see the CertificateOptionControllerTests.cs or CertificateVersionControllerTests.cs

Can we restructure the check controller tests into that format? Do you prefer this format? or the one i've suggested above, happy to talk it through